### DISCUSSION TITLE
RFC: Feedback and suggestions

### DISCUSSION THREAD

**Discussion by mvllow** [rose-pine/neovim#195] [General]
Would love to gather any feedback for the recent changes in v2, including a dedicated `styles.transparency` option. Curious what the community wants out of a transparency mode and how we can improve certain behaviours.

https://github.com/rose-pine/neovim/releases/tag/v2.0.0

**Comment by HermesSantos** [comment]
Can we have a tmux colors based?

**Reply by mvllow** [reply]
We do have a tmux theme if that's what you are requesting: https://github.com/rose-pine/tmux
